### ISSUE TITLE
malformed validation scenarios fail silently when running for Job

### ISSUE THREAD

**Issue by ghukill** [MI-DPLA/combine#431]
Schematron based validation scenario that has an XML error with the schematron, fails silently when applied and running for a Job.

However, when testing that scenario against a single record in Combine (via the GUI), the error is apparent, e.g.:

```
Opening and ending tag mismatch: rule line 33 and pattern, line 34, column 12 (file_3fb170911b114205a51b6a7ecdc5484b.sch, line 34)
```

Would be beneficial to inform user that a Validation Scenario was erroring for records when running for a Job, instead of failing quietly.

Thoughts/Questions:

  * perhaps the "validation" tab in a Job details page should show errors, in addition to valid/invalid?
  * should a Job get marked as **invalid** when errors are reported to be conservative with thinking a Job / Records are valid?  (tried applying validation, didn't work, assume we don't *know* about the validity of the records)
  * how to bubble up errors from Spark?
    * each time a Record fails a validation, a corresponding `RecordValidation` instance is created
    * wonder if this could record the error that occurred during validation?  treat the Record as if it failed validation (and in a sense it did), and instead of recording why the validation failed, record the error as best as possible?

**Comment by ghukill** [comment]
Not yet tested with:
  * python code snippet validation scenarios
  * Elasticsearch query validation scenarios

**Comment by estelendur** [comment]
I'm not entirely sure this is part of this ticket? If not, I can split it out:
![image](https://user-images.githubusercontent.com/10409866/60459746-f75cb000-9c0f-11e9-94c6-b477b0f7f8af.png)

**Comment by estelendur** [comment]
YASS

![image](https://user-images.githubusercontent.com/10409866/60742040-a8798800-9f39-11e9-9454-f4f4bd8fb652.png)

**Comment by estelendur** [comment]
Resolves #431
